### ISSUE TITLE
Certain tests are locale dependent

### ISSUE THREAD

**Issue by bracki** [dropwizard/dropwizard#32]
```
jbrauer@werner:~/projects/dropwizard (master)$ cat /home/jbrauer/projects/dropwizard/dropwizard-core/target/surefire-reports/com.yammer.dropwizard.config.tests.ConfigurationFactoryTest.txt
-------------------------------------------------------------------------------
Test set: com.yammer.dropwizard.config.tests.ConfigurationFactoryTest
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: -2,104,463.334 sec <<< FAILURE!
throwsAnExceptionOnInvalidFiles(com.yammer.dropwizard.config.tests.ConfigurationFactoryTest)  Time elapsed: 0 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: a string ending with "factory-test-invalid.yml has the following errors:\n  * name must match \"[\w]+[\s]+[\w]+\" (was Boop)"
     got: "/home/jbrauer/projects/dropwizard/dropwizard-core/target/test-classes/factory-test-invalid.yml has the following errors:\n  * name muss auf Ausdruck \"[\w]+[\s]+[\w]+\" passen (was Boop)"

    at org.junit.Assert.assertThat(Assert.java:780)
    at org.junit.Assert.assertThat(Assert.java:738)
    at com.yammer.dropwizard.config.tests.ConfigurationFactoryTest.throwsAnExceptionOnInvalidFiles(ConfigurationFactoryTest.java:58)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
    at org.junit.runners.Suite.runChild(Suite.java:128)
    at org.junit.runners.Suite.runChild(Suite.java:24)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
    at org.apache.maven.surefire.junitcore.ConfigurableParallelComputer$AsynchronousRunner$1.call(ConfigurableParallelComputer.java:186)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)

jbrauer@werner:~/projects/dropwizard (master)$ cat /home/jbrauer/projects/dropwizard/dropwizard-core/target/surefire-reports/com.yammer.dropwizard.validation.tests.ValidatorTest.txt
-------------------------------------------------------------------------------
Test set: com.yammer.dropwizard.validation.tests.ValidatorTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: -2,104,463.747 sec <<< FAILURE!
returnsASetOfErrorsForAnObject(com.yammer.dropwizard.validation.tests.ValidatorTest)  Time elapsed: 0.001 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is <[notNull may not be null (was null), tooBig must be less than or equal to 30 (was 50)]>
     got: <[notNull kann nicht null sein (was null), tooBig muss kleinergleich 30 sein (was 50)]>

    at org.junit.Assert.assertThat(Assert.java:780)
    at org.junit.Assert.assertThat(Assert.java:738)
    at com.yammer.dropwizard.validation.tests.ValidatorTest.returnsASetOfErrorsForAnObject(ValidatorTest.java:34)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
    at org.junit.runners.Suite.runChild(Suite.java:128)
    at org.junit.runners.Suite.runChild(Suite.java:24)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
    at org.apache.maven.surefire.junitcore.ConfigurableParallelComputer$AsynchronousRunner$1.call(ConfigurableParallelComputer.java:186)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)
```

**Comment by liuzhongshu** [comment]
I find some tests failed like this.  So i can only run mvn test -DargLine="-Duser.language=en -Duser.region=US" to get around this issue.

**Comment by joschi** [comment]
@liuzhongshu Could you please point out which tests were failing and which locale you were using? I'm using `de_DE` as my locale and all tests are passing.

**Comment by liuzhongshu** [comment]
@joschi Sorry for late, I am using zh_CH locale and testing dropwizard 0.6.2, and get following:

Failed tests:   throwsAnInvalidEntityExceptionForPartialValidatedRequestEntities
(com.yammer.dropwizard.jersey.tests.JacksonMessageBodyProviderTest): expecting:(
..)
  throwsAnInvalidEntityExceptionForInvalidRequestEntities(com.yammer.dropwizard.
jersey.tests.JacksonMessageBodyProviderTest): expecting:(..)
  rejectsPrivilegedPorts(com.yammer.dropwizard.validation.tests.PortRangeValidat
orTest): expecting:(..)
  rejectsNegativePorts(com.yammer.dropwizard.validation.tests.PortRangeValidator
Test): expecting:(..)
  allowsForCustomMaximumPorts(com.yammer.dropwizard.validation.tests.PortRangeVa
lidatorTest): expecting:(..)
  allowsForCustomMinimumPorts(com.yammer.dropwizard.validation.tests.PortRangeVa
lidatorTest): expecting:(..)

**Comment by joschi** [comment]
The issues will be addressed in #1056.
